web/gui/dashboard_info.js

```diff
@@ -4148,6 +4148,61 @@ netdataDashboard.context = {
         info: '<p>Network traffic received and sent by pgbouncer.</p><p><b>Received</b> - received from clients. <b>Sent</b> - sent to servers.</p>'
     },
 
+    // ------------------------------------------------------------------------
+    // CASSANDRA
+
+    'cassandra.client_requests_rate': {
+        info: 'Client requests received per second. Consider whether your workload is read-heavy or write-heavy while choosing a compaction strategy.'
+    },
+    'cassandra.client_requests_latency': {
+        info: 'Response latency of requests received per second. Latency could be impacted by disk access, network latency or replication configuration.'
+    },
+    'cassandra.cache_hit_ratio': {
+        info: 'Key cache hit ratio indicates the efficiency of the key cache. If ratio is consistently < 80% consider increasing cache size.'
+    },
+    'cassandra.cache_hit_rate': {
+        info: 'Key cache hit rate measures the cache hits and misses per second.'
+    },
+    'cassandra.storage_live_disk_space_used': {
+        info: 'Amount of live disk space used.'
```

Comment:
The [chart title](https://github.com/netdata/go.d.plugin/blob/9edeacf3fe4b0fa41bae03215b029e6dbb1890c0/modules/cassandra/charts.go#L136) is "Disk space used by live data".

I think "Amount of live disk space used." is less descriptive. Consider finding something better.

Reply:
At least you can elaborate on what live data is.

Reply:
ping @shyamvalsan

Reply:
e.g. add "This does not include obsolete data waiting to be garbage collected."